src/main/java/duke/Event.java

```diff
@@ -0,0 +1,18 @@
+package duke;
+
+import java.time.LocalDate;
+
+/**
+ * Represents an {@code Event} object. Inherits from {@code Task} object

```

Comment:
Maybe you could add a period (".") after all sentences in longer Javadoc comments? So that punctuation is consistent and the generated Javadocs would look more complete/professional. ^^ (To be clear, this refers to other Javadoc comments as well~)

Reply:
Will surely do! Thanks for the input.